layouts/section/project.html

```diff
@@ -0,0 +1,30 @@
+{{- define "main" -}}
+
+{{ partial "page_header.html" . }}
+
+<div class="universal-wrapper">
+
+  {{ with .Content }}
+  <div class="article-style">{{ . }}</div>
+  {{ end }}
+
+  {{ $paginator := .Paginate .Data.Pages }}
```

Comment:
Since we split the current and past project, it makes sense to remove the pagination from this page

Reply:
This should be resolved now (before previous meeting)